src/quequations.jl

```diff
@@ -26,11 +39,14 @@ function liouvillian_op(h::QuBase.AbstractQuMatrix)
                     sm = (n-1)*nb + m
                     sj = (j-1)*nb + i
                     lv = zero(Complex128)
+                    for l=1:nlop
+                        lv = lv + coeffs(collapse_ops[l])[m,i]*coeffs(collapse_ops[l]')[n,j]
```

Comment:
I think you don't need `coeffs` here and below. QuArrays should support `getindex`.